File: JSBASIC/70_is_consonant.js
// 70	70_is_consonant.js	Check whether a given character is a consonant.

let ch = 'b';
let vowels = "aeiouAEIOU";
console.log("Character:", ch);
if (ch.match(/[a-zA-Z]/) && !vowels.includes(ch)) {
    console.log(ch, "is a Consonant.");
} else {
    console.log(ch, "is NOT a Consonant.");
}

File: JavaBasic/70_is_consonant.java
// 70	70_is_consonant.java	Check whether a given character is a consonant.
public class _70_is_consonant {
    public static void main(String[] args) {
        char ch = 'b';
        String vowels = "aeiouAEIOU";
        System.out.println("Character: " + ch);
        if (Character.isLetter(ch) && vowels.indexOf(ch) == -1)
            System.out.println(ch + " is a Consonant.");
        else
            System.out.println(ch + " is NOT a Consonant.");
    }
}
